app/styles/page-not-found.css

```diff
@@ -0,0 +1,13 @@
+.page-not-found{
+    margin: auto;
+    text-align: center;
+  }
+  
+  .page-not-found__img {
+    width: 20%;

```

Comment:
We can maybe use `px` because in the video demonstration, I can see the image becoming too small.
Add media query for mobile view if necessary